### PR TITLE
Work on the move to Rspamd

### PR DESCRIPTION
**Pull request by stephdl**
This is a work on the move to rspamd, amavisd ans spamassassin wil be removed

NethServer/dev#5394

### CODE REVIEW

File: nethserver-mail-common.spec [hunk 1]
```diff
@@ -8,7 +8,8 @@ Source0: %{name}-%{version}.tar.gz
 BuildArch: noarch
 
 Requires: nethserver-mail-smarthost
-Requires: amavisd-new, perl-Convert-BinHex, tmpwatch
+Requires: perl-Convert-BinHex, tmpwatch
```
Comment (DavidePrincipi) [inline nethserver-mail-common.spec]: From the git history I see those are `amavisd` deps: remove them

File: nethserver-mail-common.spec [hunk 2]
```diff
@@ -8,7 +8,8 @@ Source0: %{name}-%{version}.tar.gz
...
+Obsoletes: amavisd-new
```
Comment (DavidePrincipi) [inline nethserver-mail-common.spec]: I wouldn't do it: uninstalling amavisd-new is not necessary. As the `prop` type default is removed, the amavisd service should be automatically disabled.

**Review by DavidePrincipi — CHANGES_REQUESTED** [review]
Automatic uninstall of amavisd-new is not necessary IMO. To fully clean up we should handle deps, too. I'd prefer to document a manual procedure for those sysadmins who like a clean RPM db.

**Note**
Rebase on `master`

**Comment by stephdl** [comment]
Really no fear to rebase on master, @DavidePrincipi, I learnt something tonight :D

File: nethserver-mail-common.spec [hunk 3]
```diff
@@ -1,6 +1,6 @@
 Summary: Common configuration for mail packages
 Name: nethserver-mail-common
-Version: 1.6.6
+Version: 1.6.7
```
Comment (DavidePrincipi) [inline nethserver-mail-common.spec]: This can be incremented only for a tagged release, after QA phase.

File: createlinks [hunk 4]
```diff
@@ -85,7 +85,7 @@ my @domainTemplates = qw(
 foreach (qw(create modify delete)) {
     event_templates('domain-' . $_, @domainTemplates);
     event_actions('domain-'   . $_, 'nethserver-mail-postmap-update' => '30');
-    event_services('domain-'  . $_, 'postfix' => 'restart');
+    event_services('domain-'  . $_, 'postfix' => 'restart','opendkim' => 'restart');
```
Comment (DavidePrincipi) [inline createlinks]: Did we forgot to do it on PR #17 ? I think it is not related to rspamd...

**Review by DavidePrincipi — CHANGES_REQUESTED** [review]
~It seems this PR is quite empty: can we close it without merging? The opendkim restart on domain events should be splitted in a new PR for issue NethServer/dev#5407~

Edit:
I was looking at the last commit only, sorry my fault

File: nethserver-mail-common.spec (hunk 3 above)
Comment (stephdl) [reply nethserver-mail-common.spec]: yep my human fault, I will remove it

File: root/usr/share/nethesis/NethServer/Language/en/NethServer_Module_Mail_Domain.php [hunk 5]
```diff
@@ -2,8 +2,6 @@
 
 $L['Create a new domain'] = 'Create a new domain';
 $L['Delete_label'] = 'Delete';
-$L['DisclaimerStatus_label'] = 'Append a legal note to sent messages';
```
Comment (DavidePrincipi) [inline root/usr/share/nethesis/NethServer/Language/en/NethServer_Module_Mail_Domain.php]: We need this line for Transifex: if two alternative packages exist the same string catalog file have to support both of them

File: nethserver-mail-common.spec [hunk 6]
```diff
@@ -9,8 +9,9 @@ BuildArch: noarch
 
 Provides: nethserver-mail-disclaimer = 0.0.0
 Requires: nethserver-mail-smarthost
-Requires: amavisd-new, perl-Convert-BinHex, tmpwatch
 Requires: opendkim
+Requires: perl-Convert-BinHex, tmpwatch
```
Comment (DavidePrincipi) [inline nethserver-mail-common.spec]: Still this line has to be removed 😒 😄

**Review by DavidePrincipi — CHANGES_REQUESTED** [review]

File: root/etc/e-smith/templates/etc/postfix/master.cf/00template_vars [hunk 7]
```diff
@@ -7,7 +7,6 @@
     @smtpd_public_options = (
 	'smtpd_helo_required=yes',
 	'strict_rfc821_envelopes=yes',
-    'smtpd_proxy_filter=127.0.0.1:10024',
 	'smtpd_proxy_options=speed_adjust',
```
Comment (DavidePrincipi) [inline root/etc/e-smith/templates/etc/postfix/master.cf/00template_vars]: Remove also `smtpd_proxy_*` everywhere

**Review by DavidePrincipi — CHANGES_REQUESTED** [review]

**Review by DavidePrincipi — APPROVED** [review]
Well done!

**Pull request merged**